src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -214,907 +214,908 @@ void GetPurchasedAir(EnergyPlusData &state)
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Total Heating Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).HtRecTotHeatEnergy,
+                            PurchAir.HtRecTotHeatEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Sensible Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).HtRecSenCoolEnergy,
+                            PurchAir.HtRecSenCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Latent Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).HtRecLatCoolEnergy,
+                            PurchAir.HtRecLatCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Total Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).HtRecTotCoolEnergy,
+                            PurchAir.HtRecTotCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Zone Heating Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.ZoneTotHeatFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Zone Cooling Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.ZoneTotCoolFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name);
```

Comment:
Added two energy consumption report variables for zones: `Zone Ideal Loads Zone Heating Fuel Energy` and `Zone Ideal Loads Cooling Fuel Energy`.

Reply:
> Added two energy consumption report variables for zones: `Zone Ideal Loads Zone Heating Fuel Energy` and `Zone Ideal Loads Cooling Fuel Energy`.

How do these differ from "Zone Ideal Loads Supply Air Total Heating Fuel Energy"?